examples/source/interactivity-dynamic-content/Comment_Section/index.html

```diff
@@ -136,11 +136,11 @@
     This sample allows an user to login and logout using an email and a password. Logout is implemented by configuring a second endpoint in the login property `sign-out`, find more [here]({{g.doc('/content/amp-dev/documentation/components/reference/amp-access.md', locale=doc.locale).url.path}}#login-page).  -->
     <script id="amp-access" type="application/json">
     {
-        "authorization": "<% hosts.backend %>/components/amp-access/authorization?rid=READER_ID&url=CANONICAL_URL&ref=DOCUMENT_REFERRER&_=RANDOM",
+        "authorization": "/documentation/examples/api/amp-access/authorization?rid=READER_ID&url=CANONICAL_URL&ref=DOCUMENT_REFERRER&_=RANDOM",
         "noPingback": "true",
         "login": {
-          "sign-in": "<% hosts.backend %>/components/amp-access/login?rid=READER_ID",
-          "sign-out": "<% hosts.backend %>/components/amp-access/logout?rid=READER_ID"
+          "sign-in": "/documentation/examples/api/amp-access/login?rid=READER_ID",
```

Comment:
What I find a bit confusing now: some component APIs ended up in `examples/source/1.components/amp-live-list/api.js` for example some in `examples/api/amp-consent.js`. I personally like the former one better as it couples logic and markup - what do you think?

Reply:
Since multiple examples (two atm) are using this backend, the general way of implementing this in the other cases was to put it somewhere central. Example specific APIs are located in the example folder.